pkg/resourcecreator/googlesqlinstance.go

```diff
@@ -19,6 +19,7 @@ const (
 	DefaultSqlInstanceAutoBackupHour = 2
 	DefaultSqlInstanceTier           = "db-f1-micro"
 	DefaultSqlInstanceDiskSize       = 10
+	DefaultSqlInstanceCollation      = "en_US.UTF8"
```

Comment:
Burde ikke default vært norsk?

Reply:
Det er en diskusjon noen andre kan bestemme, jeg føler ikke sterkt for hverken eller.

Mitt valg var bestemt av tanken "bakoverkompatibilitet" - i tilfelle noe kunne knekke for neste database for team X.

Reply:
Enig med @Kyrremann på bruk av norsk, for en gangs skyld.
Her er character set UTF-8, som bør være bakoverkompatibelt og dekker alle tegn. Så har man sorteringsrekkefølge, som til fordel kan være norsk. Hvis det er tvil om dette kan man jo fyre av et spørsmål i en av Slack-kanalene.